addons/website_sale/static/src/snippets/s_add_to_cart/options.xml

```diff
@@ -0,0 +1,34 @@
+<templates id="template" xml:space="preserve">
+<t t-name="website_sale.s_add_to_cart_options">
+    <WeRow>
+        <t t-set-slot="title">Product</t>
+        <WeMany2one model="'product.template'"
+                    setProductTemplate="''"
+                    name="'product_template_picker_opt'"
+                    noPreview="'true'"
+                    domain='`[["is_published", "=", true], ["sale_ok", "=", true]]`'/>
+        <WeButton name="'product_template_reset_opt'"
+                  class="'reset-product-picker align-self-end fa fa-fw fa-times'"
+                  t-on-click="() => env.snippetOption._onClickResetProductPicker()"/>
+    </WeRow>
+    <WeRow>
+        <t t-set-slot="title">Variant</t>
+        <t t-set="defaultMessage">Visitor's Choice</t>
+        <WeMany2one class="'o_we_sublevel_1'"
+                    model="'product.product'"
+                    setProductVariant="''"
+                    name="'product_variant_picker_opt'"
+                    noPreview="'true'"
+                    defaultMessage="defaultMessage"
+                    domain="renderContext.variantDomain"/>
+        <WeButton name="'product_variant_reset_opt'"
+                  class="'reset-variant-picker align-self-end fa fa-fw fa-times'"
+                  t-on-click="() => env.snippetOption._onClickResetVariantPicker()"/>
```

Comment:
I cherry-picked anyway, but it would be cool to change this to a simple props + option method. But it was already like this prior to OWL so... 🤷🏻